package.json

```diff
@@ -31,11 +31,12 @@
     "execa": "^7.2.0"
   },
   "devDependencies": {
-    "@types/jest": "29.5.14",
+    "@types/jest": "30.0.0",
     "@vercel/ncc": "0.38.3",
-    "jest": "29.7.0",
+    "jest": "30.1.3",
     "prettier": "3.6.2",
     "ts-jest": "29.4.1",
+    "ts-node": "^10.9.2",
```

Comment:
❓ What do we need the ts-node dependency for? Is it just to run jest.config.ts directly? Can we transpile it to js instead?

I'm asking because I'm actually eager to reduce the number of dependencies rather than add new ones :D

Reply:
Indeed, `ts-node` is required for Jest to transpile it's own config. Having it in TypeScript ensures configuration is correct (and you have a nice auto-complete in IDE), but I agree it might not be as crucial as adding an extra dependency.

Reply:
Thanks @yafanasiev, but how come `npm test` locally still works after removing the ts-node dependency?

Do you see a way that allows typescript to transpile it somehow? Or is that not possible?

Alternatively, are there other benefits we can get from depending on ts-node?

Reply:
huh, that's very weird. It worked for me after removing `ts-node` as well, but now in the CI it fails with the expected error. I played locally and it might be that you are using NodeJS v24 locally, which now supports native TS transpiling, as when I switched to Node v20 it failed as expected.